photon-lib/src/main/native/cpp/photonlib/PhotonCamera.cpp

```diff
@@ -134,10 +134,29 @@ void PhotonCamera::VerifyVersion() {
   const std::string& versionString = versionEntry.Get("");
   if (versionString.empty()) {
     std::string path_ = path;
-    FRC_ReportError(
-        frc::warn::Warning,
-        "PhotonVision coprocessor at path {} not found on NetworkTables!",
-        path_);
+    std::vector<std::string> cameraNames =
```

Comment:
Actually. this doesn't work locally for me. You'll only get back a vector of subtables your client has subscribed to, it seems like? 
![image](https://user-images.githubusercontent.com/29715865/210669833-45ac21a2-f142-4c30-a724-475cf44210e0.png)

Reply:
MVP shown here: https://cdn.discordapp.com/attachments/368993897495527424/1060342579381669978/image.png

Reply:
@mcm001 Based on the screenshot, is it possible that the `photonvision` table wasn't initialized yet when you made those calls?

Reply:
Correct, this is an NT4ism.

Our implementation should first subscribe to `/photonvision/*` with the "topic names only" flag set. Then downstream things should work.